index.js

```diff
@@ -310,6 +327,10 @@ const ContainerService = {
   },
 
   sortTabs() {
+    sendEvent({
```

Comment:
If we use this approach, we can easily call: this._sendTelemetryData in other parts of the code.